readme.md

```diff
@@ -760,6 +760,7 @@
 
 - [Slack](https://github.com/matiassingers/awesome-slack#readme) - Team collaboration.
 	- [Communities](https://github.com/filipelinhares/awesome-slack#readme)
+	- [Slack Apps For Developers](https://github.com/axolo-co/awesome-slack-apps) - List of Slack integration to help developers and tech teams be more productive.
```

Comment:
Contribution guide says to link directly to the README, and the tagline should be a description of the topic, not a description of the list.
```suggestion
	- [Slack Apps For Developers](https://github.com/axolo-co/awesome-slack-apps#readme) - Slack integrations to help developers and tech teams be more productive.
```